### DISCUSSION TITLE
[iOS] `FullWindowOverlay` view is obliterated when a `fullScreenModal` is rendered

### DISCUSSION THREAD

**Discussion by awlui** [software-mansion/react-native-screens#1777] [Q&A] [answered]
I'm trying to render toast notifications above everything else, including full screen modals. `FullWindowOverlay` seemed like a good candidate but it seems to get removed just like everything else once a full screen modal is triggered. Any suggestions?

**Comment by totodot** [comment]
Same here, for me changing `fullScreenModal` to `transparentModal` helps.

**Comment by tboba** [comment, marked as answer]
Hi @awlui, thanks for creating the thread!
Yeah, we're aware of the problem you've mentioned (apparently, someone has reported similar problem with the toasts from `Burnt` library. We're planning to fix that problem in the near future.
You can find more about it [here](https://github.com/software-mansion/react-native-screens/issues/1930).

Cheers!
